src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0,
+        minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="logo"
+          class="image_photo"
+        >
+      </div>
+
+      <div class="card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+
+      <div class="card__code">
+        Код товара: 195434
+      </div>
+
+      <div class="rating card__rating">
+
+        <div class="stars rating__stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="rating__reviews">
+          Отзывов: 5
+        </div>
+      </div>
+
+      <div class="price-list card__price-list">
+
+        <div class="price-list__prise">
+          Цена:
+        </div>
+
+        <div class="price-list__how-much">
+          69 999 грн
+        </div>
+      </div>
+
+      <div class="link card__link">
+        <a
+          href="#"
+          class="link__bay"
```

Comment:
```suggestion
          class="card__link"
```
Let's make it an element of `card`